client-v2/src/shared/components/BasePlaceholder.tsx

```diff
@@ -13,12 +13,12 @@ injectGlobal`
 `;
 
 export default styled('div')`
```

Comment:
I think this, and possibly other files, are still using the base `styled` function -- difficult to spot as there's no red ink, but this will mean that .props.theme is `any` typed! If that's right, it'd be worth replacing anything that uses the base `styled` function before merging.

Reply:
Good catch! This has been fixed in the shared components. It might be something to flag in the readme? I have also added it as a note in the Theme.ts file.

Reply:
Ace! `flag in the readme` -- great idea.